psi4/driver/procrouting/dft_funcs/dict_xc_funcs.py

```diff
@@ -0,0 +1,153 @@
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# Psi4 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Psi4; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# @END LICENSE
+#
+"""
+List of XC functionals
+"""
+
+import copy
+
+funcs = []
+
+# yapf: disable
+funcs.append({"name": "TETER93"        , "xc_functionals": {"LDA_XC_TETER93"            : {}}})
+funcs.append({"name": "ZLP"            , "xc_functionals": {"LDA_XC_ZLP"                : {}}})
+funcs.append({"name": "KSDT"           , "xc_functionals": {"LDA_XC_KSDT"               : {}}})
+funcs.append({"name": "OPBE-D"         , "xc_functionals": {"GGA_XC_OPBE_D"             : {}}, "dispersion": {"type": "d2gr", "params": {'s6': 1.0,  'alpha6': 20.0, 'rs6': 1.15}, "citation": '    L. Goerigk, S. Grimme, J. Chem. Theory. Comput. 6, 107-126, 2010'}})
+funcs.append({"name": "OPWLYP-D"       , "xc_functionals": {"GGA_XC_OPWLYP_D"           : {}}, "dispersion": {"type": "d2gr", "params": {'s6': 1.0,  'alpha6': 20.0, 'rs6': 1.15}, "citation": '    L. Goerigk, S. Grimme, J. Chem. Theory. Comput. 6, 107-126, 2010'}})
+funcs.append({"name": "OBLYP-D"        , "xc_functionals": {"GGA_XC_OBLYP_D"            : {}}, "dispersion": {"type": "d2gr", "params": {'s6': 1.0,  'alpha6': 20.0, 'rs6': 1.15}, "citation": '    L. Goerigk, S. Grimme, J. Chem. Theory. Comput. 6, 107-126, 2010'}})
+funcs.append({"name": "HCTH/407P"      , "xc_functionals": {"GGA_XC_HCTH_407P"          : {}}, "alias": ["HCTH407P"]})
+funcs.append({"name": "HCTH/P76"       , "xc_functionals": {"GGA_XC_HCTH_P76"           : {}}, "alias": ["HCTHP76"]})
+funcs.append({"name": "HCTH/P14"       , "xc_functionals": {"GGA_XC_HCTH_P14"           : {}}, "alias": ["HCTHP14"]})
+funcs.append({"name": "B97-GGA1"       , "xc_functionals": {"GGA_XC_B97_GGA1"           : {}}})
+funcs.append({"name": "KT2"            , "xc_functionals": {"GGA_XC_KT2"                : {}}})
+funcs.append({"name": "TH1"            , "xc_functionals": {"GGA_XC_TH1"                : {}}})
+funcs.append({"name": "TH2"            , "xc_functionals": {"GGA_XC_TH2"                : {}}})
+funcs.append({"name": "TH3"            , "xc_functionals": {"GGA_XC_TH3"                : {}}})
+funcs.append({"name": "TH4"            , "xc_functionals": {"GGA_XC_TH4"                : {}}})
+funcs.append({"name": "HCTH/93"        , "xc_functionals": {"GGA_XC_HCTH_93"            : {}}, "alias": ["HCTH", "HCTH93"]})
+funcs.append({"name": "HCTH/120"       , "xc_functionals": {"GGA_XC_HCTH_120"           : {}}, "alias": ["HCTH120"], })
```

Comment:
Your right, the "/" won't work. I think the most canonical thing would be to just remove the "/" here?

Reply:
I was trying to trace through the CBS parsing, and I don't see anything fancy enough to allow the `HCTH/` through unscathed. Maybe `HCTH-P14` since there's adjacent letters for some variants.

Excellent catch. I think we already have `sapt2+(3)` confusing CBS — don't need more.